{"identifier":{"url":"doc:\/\/SwiftDirector\/documentation\/SwiftDirector\/LDAPError\/unknown","interfaceLanguage":"swift"},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"unknown"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"LDAPError","preciseIdentifier":"s:13SwiftDirector9LDAPErrorV","identifier":"doc:\/\/SwiftDirector\/documentation\/SwiftDirector\/LDAPError"},{"kind":"text","text":" { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"platforms":["Linux"]}]}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftdirector\/ldaperror\/unknown"]}],"hierarchy":{"paths":[["doc:\/\/SwiftDirector\/documentation\/SwiftDirector","doc:\/\/SwiftDirector\/documentation\/SwiftDirector\/LDAPError"]]},"metadata":{"role":"symbol","roleHeading":"Type Property","title":"unknown","modules":[{"name":"SwiftDirector"}],"symbolKind":"property","externalID":"s:13SwiftDirector9LDAPErrorV7unknownACvpZ","fragments":[{"text":"static","kind":"keyword"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"unknown"},{"kind":"text","text":": "},{"preciseIdentifier":"s:13SwiftDirector9LDAPErrorV","kind":"typeIdentifier","text":"LDAPError"}]},"kind":"symbol","schemaVersion":{"minor":2,"major":0,"patch":0},"abstract":[{"text":"An unknown error that does not correspond to any LDAP error.","type":"text"},{"text":" ","type":"text"},{"text":"Usually occurs if ldap returned a success status code but not the necessary results.","type":"text"}],"sections":[],"references":{"doc://SwiftDirector/documentation/SwiftDirector":{"kind":"symbol","title":"SwiftDirector","abstract":[],"type":"topic","identifier":"doc:\/\/SwiftDirector\/documentation\/SwiftDirector","url":"\/documentation\/swiftdirector","role":"collection"},"doc://SwiftDirector/documentation/SwiftDirector/LDAPError/unknown":{"identifier":"doc:\/\/SwiftDirector\/documentation\/SwiftDirector\/LDAPError\/unknown","type":"topic","kind":"symbol","abstract":[{"type":"text","text":"An unknown error that does not correspond to any LDAP error."},{"text":" ","type":"text"},{"text":"Usually occurs if ldap returned a success status code but not the necessary results.","type":"text"}],"fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"unknown"},{"kind":"text","text":": "},{"preciseIdentifier":"s:13SwiftDirector9LDAPErrorV","kind":"typeIdentifier","text":"LDAPError"}],"title":"unknown","url":"\/documentation\/swiftdirector\/ldaperror\/unknown","role":"symbol"},"doc://SwiftDirector/documentation/SwiftDirector/LDAPError":{"type":"topic","url":"\/documentation\/swiftdirector\/ldaperror","kind":"symbol","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"LDAPError"}],"identifier":"doc:\/\/SwiftDirector\/documentation\/SwiftDirector\/LDAPError","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"LDAPError"}],"title":"LDAPError","abstract":[{"type":"text","text":"The error type representing errors occurring during LDAP operations."}]}}}